python/cuopt_self_hosted/cuopt_sh_client/cuopt_sh.py

```diff
@@ -587,6 +592,15 @@ def main():
         help="Upload a solution to be cached on the server. The reqId "
         "returned may be used as an initial solution for VRP.",
     )
+    parser.add_argument(
+        "-ht",
+        "--http-timeout",
+        type=int,
+        default=30,
```

Comment:
Think it is better to set None, since the possibility of hitting this issue might be higher